From 3be79a564343886ade7e88f447944fd4b5452959 Mon Sep 17 00:00:00 2001
From: Craig Andrews <candrews@integralblue.com>
Date: Fri, 31 Jul 2020 14:56:21 -0400
Subject: [PATCH] [test] [webserver] Conditional TestHTTPDirectory

TestHTTPDirectory.cpp requires microhttpd so only include it if microhttpd is found.

TestHTTPDirectory.cpp includes network/WebServer.h which includes network/httprequesthandler/IHTTPRequestHandler.h which includes microhttpd.h
---
 xbmc/filesystem/test/CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/xbmc/filesystem/test/CMakeLists.txt b/xbmc/filesystem/test/CMakeLists.txt
index 235acc8549cf..9572459cf198 100644
--- a/xbmc/filesystem/test/CMakeLists.txt
+++ b/xbmc/filesystem/test/CMakeLists.txt
@@ -1,10 +1,13 @@
 set(SOURCES TestDirectory.cpp
             TestFile.cpp
             TestFileFactory.cpp
-            TestHTTPDirectory.cpp
             TestZipFile.cpp
             TestZipManager.cpp)
 
+if(MICROHTTPD_FOUND)
+  list(APPEND SOURCES TestHTTPDirectory.cpp)
+endif()
+
 if(NFS_FOUND)
   list(APPEND SOURCES TestNfsFile.cpp)
 endif()
